specs/latest/1.0/index.html

```diff
@@ -453,13 +453,13 @@ <h2 class="no-toc">Table of contents</h2>
         textures, buffers (i.e., VBOs), framebuffers, renderbuffers, shaders and
         programs. The WebGLRenderingContext interface has a method to create a
         WebGLObject subclass for each type. Data from the underlying graphics library are
-        stored in these objects and are fully managed by them. The resources represented
-        by these objects are guaranteed to exist as long as the object exists.
-        Furthermore, the DOM object is guaranteed to exist as long as the author has an explicit
-        valid reference to it <b>or</b> as long as it is bound by the underlying graphics library.
-        When none of these conditions exist the user agent can, at any point, delete the object
-        using the equivalent of a delete call (e.g., deleteTexture). If authors wish to control
-        when the underlying resource is released then the delete call can be made explicitly.
+        stored in these objects and are fully managed by them.
+        The DOM object is guaranteed to exist at least as long as the author has an explicit
+        valid reference to it <b>and</b> at least as long as it is bound by the underlying graphics
+        library.
```

Comment:
Re-reading, I see your point, but think the original text was more clear. Can I suggest:

The DOM object is guaranteed to exist while either the author has an explicit valid reference to it, or it is bound by the underlying graphics library.